webui/react/src/components/Table/Table.tsx

```diff
@@ -200,20 +200,36 @@ export const experimentProgressRenderer: ExperimentRenderer = (_, record) => {
 
 /* Model Table Column Renderers */
 
-export const modelNameRenderer = (value: string, record: ModelItem): React.ReactNode => (
-  <Space className={css.wordBreak}>
-    <div style={{ paddingInline: 4 }}>
-      <Icon name="model" size="medium" />
-    </div>
-    <Link path={paths.modelDetails(String(record.id))}>{value}</Link>
-  </Space>
-);
+export const modelNameRenderer = (
+  value: string,
+  record: ModelItem,
+  workspaceId?: number,
```

Comment:
from yesterday we will record `model.workspace_id` from the API, would you want to use that and remove workspaceId as a separate param here?

Reply:
Thanks for the info!